app/1.0/start/toolbox/create-a-page.md

```diff
@@ -29,125 +27,85 @@ your new view.
         <style>
           :host {
             display: block;
-
             padding: 16px;
           }
         </style>
-
         <h1>New view</h1>
       </template>
       <!-- Creates the element's prototype and registers it -->
       <script>
         Polymer({
-          is: 'my-new-view'
+          is: 'my-new-view',
+          properties: {
+            route: Object
+          }
         });
       </script>
     </dom-module>
-
     ```
 
-For now your element is very basic, and just has a `<h1>` that says "New View",
-but we can return to it and make it more interesting later.
+## Add the page element to your app
 
-## Add the element to your app
+Your element is defined, but your app isn't actually using it yet. To use it,
+you need to declare it in your app's HTML.
 
 1.  Open `src/my-app.html` in a text editor.
 
-1.  Find the set of existing pages inside the `<iron-pages>`:
-
-    ```
-    <iron-pages
-        selected="[[page]]"
-        attr-for-selected="name"
-        fallback-selection="view404"
-        role="main">
-      <my-view1 name="view1"></my-view1>
-      <my-view2 name="view2"></my-view2>
-      <my-view3 name="view3"></my-view3>
-      <my-view404 name="view404"></my-view404>
-    </iron-pages>
-    ```
-
-    The `<iron-pages>` is bound to the `page` variable that changes with the
-    route, and selects the active page while hiding the others.
+1.  Find the `iron-pages` element and declare your new page at the bottom. 
 
-1.  Add your new page inside the iron-pages:
 
     ```
-    <my-new-view name="new-view"></my-new-view>
+      <iron-pages role="main" selected="[[page]]" attr-for-selected="name">
+        <my-view1 name="view1"></my-view1>
+        <my-view2 name="view2"></my-view2>
+        <my-view3 name="view3"></my-view3>
+        <my-new-view name="new-view"></my-new-view>
+      </iron-pages>
     ```
 
-    Your `<iron-pages>` should now look like this:
+    The `iron-pages` element is a reusable component that will not be 
```

Comment:
Maybe condense first two sentences? Something like, "The `iron-pages` element manages the display of its child elements, or "pages", so that only one is visible at a given time."?